### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by chokobole**
# Description

This PR updates roadmap and GPU columns in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ Symbol Definitions:
 
 - [x] 2024Q1 - Enable producing the [zkEVM](https://github.com/kroma-network/zkevm-circuits) proof.
 - [ ] 2024Q2 - Replace Halo2 with Tachyon in [Kroma](https://kroma.network/) mainnet.
-- [ ] 2024Q3 - Implement Halo2 GPU.
-- [ ] 2024Q4 - Implement Halo2 Folding Scheme.
+- [ ] 2024Q3 - Implement and optimize Plonky3 in Tachyon.
+- [ ] 2024Q3 - Replace zkEVM with zkVM in [Kroma](https://kroma.network/) mainnet.
```
Comment (ashjeong) [inline README.md:87]: Not sure if this is a better change 🤔, but I'll leave the suggestion
```suggestion
- [ ] 2024Q3a - Implement and optimize Plonky3 in Tachyon.
- [ ] 2024Q3b - Replace zkEVM with zkVM in [Kroma](https://kroma.network/) mainnet.

Comment (TomTaehoonKim) [inline README.md:87]: Might be too verbose, but to justify the implementation of Plonky3 in Tachyon and alignment with zkVM, how about specifying zkVM as SP1 zkVM?

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Review by ashjeong — COMMENTED** [review]
LGTM

**Comment by ashjeong** [comment]
Wondering on the side, but although we have Halo2 added as a Lookup should we change this to Halo2 Lookup and add Halo2 Log Derivative Lookup?

**Comment by chokobole** [comment]
> Wondering on the side, but although we have Halo2 added as a Lookup should we change this to Halo2 Lookup and add Halo2 Log Derivative Lookup?

Since we don't have C api and rust binding for this, what if updating the table after this?

File: README.md [hunk 2]
```diff
@@ -83,8 +83,8 @@ Symbol Definitions:
 
 - [x] 2024Q1 - Enable producing the [zkEVM](https://github.com/kroma-network/zkevm-circuits) proof.
 - [ ] 2024Q2 - Replace Halo2 with Tachyon in [Kroma](https://kroma.network/) mainnet.
-- [ ] 2024Q3 - Implement Halo2 GPU.
-- [ ] 2024Q4 - Implement Halo2 Folding Scheme.
+- [ ] 2024Q3a - Implement and optimize Plonky3 in Tachyon.
+- [ ] 2024Q3b - Replace zkEVM with [SP1](https://github.com/succinctlabs/sp1) zkVM in [Kroma](https://kroma.network/) mainnet.
```
Comment (dongchangYoo) [inline README.md:87]: For convenience, how about marking it for Q2 and merging this PR after this Friday?

**Review by dongchangYoo — CHANGES_REQUESTED** [review]

**Comment by ashjeong** [comment]
> > Wondering on the side, but although we have Halo2 added as a Lookup should we change this to Halo2 Lookup and add Halo2 Log Derivative Lookup?
> 
> Since we don't have C api and rust binding for this, what if updating the table after this?

Sounds good~

**Review by ashjeong — APPROVED** [review]
LGTM

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Pull request merged**